af-plans/README.md

```diff
@@ -0,0 +1,10 @@
+Automation Framework Plans
+==========================
```

Comment:
Use markdown for the heading?

(This probably renders markdownishly but I mean using hash marks 😉)

Reply:
I copied the other READMEs 😉 
I'd prefer to stick with consistancy...

Reply:
That’s fine I hadn’t checked